addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1905,6 +1905,63 @@ options.registry.collapse = options.Class.extend({
     },
 });
 
+options.registry.MasonryLayout = options.Class.extend({
+    xmlDependencies: (options.Class.xmlDependencies || []).concat(
+        ['/website/static/src/snippets/s_masonry_block/s_masonry_block.xml']
+    ),
+
+    /**
+     * @override
+     */
+    start: function () {
```

Comment:
For all your function declaration, use ES6 syntax

```javascript
start() {
```

No need of the `function` keyword.